pylink/library.py

```diff
@@ -138,6 +138,22 @@ def get_appropriate_windows_sdk_name(cls):
         else:
             return Library.WINDOWS_32_JLINK_SDK_NAME
 
+    @classmethod
+    def can_load_library(cls, dllpath):
+        """Test whether a library is the correct architecture to load.
+
+        Args:
+          dllpath: A path to a library.
+
+        Returns:
+          True if the library could be successfully loaded, False if not.
+        """
+        try:
+            ctypes.CDLL(dllpath)
+            return True
+        except:
```

Comment:
Is there a specific error we could catch here instead of a raw `except`?

Reply:
Let's try catching `OSError`, that's what we get in this scenario